examples/ex18.cpp

```diff
@@ -188,38 +184,32 @@ int main(int argc, char *argv[])
 
    // 7. Set up the nonlinear form corresponding to the DG discretization of the
    //    flux divergence, and assemble the corresponding mass matrix.
-   MixedBilinearForm Aflux(&dfes, &fes);
-   Aflux.AddDomainIntegrator(new TransposeIntegrator(new GradientIntegrator()));
-   Aflux.Assemble();
-
-   NonlinearForm A(&vfes);
-   RiemannSolver rsolver;
-   A.AddInteriorFaceIntegrator(new FaceIntegrator(rsolver, dim));
-
-   // 8. Define the time-dependent evolution operator describing the ODE
-   //    right-hand side, and perform time-integration (looping over the time
-   //    iterations, ti, with a time-step dt).
-   FE_Evolution euler(vfes, A, Aflux.SpMat());
+   EulerFlux flux(dim, specific_heat_ratio);
+   RiemannSolver *numericalFlux = new RusanovFlux(flux);
+   DGHyperbolicConservationLaws euler(vfes,
+                                      new HyperbolicFormIntegrator(flux, *numericalFlux, IntOrderOffset));
```

Comment:
```suggestion
   RusanovFlux numericalFlux(flux);
   DGHyperbolicConservationLaws euler(
      vfes, new HyperbolicFormIntegrator(flux, numericalFlux, IntOrderOffset));
```
And can remove the `delete numericalFlux;` below.